app/uk/gov/hmrc/tariffclassificationfrontend/connector/FileStoreConnector.scala

```diff
@@ -64,6 +63,7 @@ class FileStoreConnector @Inject()(appConfig: AppConfig, http: HttpClient, ws: W
     )
 
     ws.url(s"${appConfig.fileStoreUrl}/file")
+      .withHeaders(http.addAuth(appConfig, hc).headers: _*)
```

Comment:
In this case we cannot use `http-verbs`, so this needs to be explicitly passed like this
We need to do the same in the other microservices (`binding-tariff-admin-frontend` + `binding-tariff-trader-frontend`) uploading files to `binding-tariff-filestore`

Reply:
Could we not just add something to `AuthenticatedHttpClient` e.g. 
`lazy val apiKeyHeader ("X-Api-Key" -> "xyz")`

Which we can call in instances like this using `http.apiKeyHeader`

Reply:
Remember that for this specific request we are not using `http-verbs`, but WsClient instead.
We could use the `wsClient` field inside  `AuthenticatedHttpClient`. I can give it a go

Reply:
do you prefer this ?
```
    ws.url(s"${appConfig.fileStoreUrl}/file")
      .withHeaders( hc.headers: _* )
      .withHeaders( http.authHeaders(appConfig) )
      .post(Source(List(filePart, dataPart)))
      .map(response => Json.fromJson[FileMetadata](Json.parse(response.body)).get)
  }
```

Reply:
Does the job :)